examples/websocket-basic-sample/websocket_basic_sample.bal

```diff
@@ -42,13 +49,20 @@ service<http:WebSocketService> basic bind { port: 9090 } {
         io:println("\nNew binary message received");
         io:print("UTF-8 decoded binary message: ");
         io:println(b);
-        caller->pushBinary(b) but { error e => log:printError("Error occurred when sending binary", err = e) };
+        caller->pushBinary(b) but {
+            error e => log:printError(
+                           "Error occurred when sending binary", err = e)
+        };
     }
 
     // This resource is triggered when a ping message is received from the client. If this resource is not implemented,
     // a pong message is automatically sent to the connected endpoint when a ping is received.
     onPing(endpoint caller, byte[] data) {
-        caller->pong(data) but { error e => log:printError("Error occurred when closing the connection", err = e) };
+        caller->pong(data) but {
+            error e => log:printError(
+                           "Error occurred when closing the connection",
+                           err = e)
```

Comment:
here also we can bring the line up I guess.

Reply:
All BBE should only contain 80 characters in a given line. Since these are longer than 80 characters, examples are not shown in the expected way in BBE.
Please take a look at https://ballerina.io/learn/by-example/websocket-basic-sample.html

Reply:
ack